server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewAssignmentServicesImpl.java

```diff
@@ -113,6 +113,10 @@ private Set<ReviewAssignment> defaultReviewAssignments(UUID reviewPeriodId) {
         Set<ReviewAssignment> reviewAssignments = new HashSet<>();
 
         memberProfileRepository.findAll().forEach(memberProfile -> {
+            if(memberProfile.getSupervisorid() == null) {
+                return;
```

Comment:
Should this return an empty set instead of returning nothing?

Reply:
Should be fine, we're just iterating over the members and conditionally adding review assignments for those w/o supervisors.